scripts/aggregate_can_data.py

```diff
@@ -0,0 +1,83 @@
+# This script recieves CAN data,sends it to FRED with MQTT,
```

Comment:
These telemetry scripts probably shouldn't be here. Note that this script sends data to Node red and writes to a CSV file so I don't think this is applicable. The aggregating data part will be but I still don't think this should be merged into master.